initfiles/etc/DIR_NAME/environment.xml.in

```diff
@@ -969,25 +969,6 @@
              name="s1"
              netAddress="."/>
   </TopoServerProcess>
-  @SPARK_ENV_S@
```

Comment:
This variable is coming from initfiles/etc/DIR_NAME/CMakeLists.txt around line 20-26?

Should remove that block as part of this PR. 

Also remove @DNG_SPARKTHOR@ from initfiles/etc/DIR_NAME/genenvrules.conf.in at the end of line 4

Reply:
Perfect thanks.